.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,37 @@
+---
+name: Bug report
+about: An issue with clap, clap_derive or clap_generate
+title: ''
+labels: 'T: bug'
+assignees: ''
+
+---
+
+<!--
+Please use the following template to assist with creating an issue and to ensure a speedy resolution. If an area is not applicable, feel free to delete the area or mark with `N/A`
+-->
+
+### Rust Version
+
+Use the output of `rustc -V`
+
+### Affected Version of `clap*`
+
+Can be found in Cargo.lock or Cargo.toml of your project (i.e. `grep clap Cargo.lock`). PLEASE DO NOT PUT "latest" HERE, use precise version. Put `master` (or other branch) if you're using the repo directly. 
+
+### Expected Behavior Summary
+
+When I do like *this*, *that* is happening and I think it shouldn't.
+
+### Actual Behavior Summary
+
+I think *this* should happen instead.
+
+### Steps to reproduce the issue
+
+1. Write *this* code '...'
+2. Run `cargo run` '....'
+3. See error
+
+### Additional context
```

Comment:
Is this blocking / is this high priority would be nice (explained below as well)